src/components/AudioPlayer/AudioProgress/AudioProgress.component.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect } from 'react';
+import propTypes from 'prop-types';
+
+import './styles/progress.scss';
+
+const AudioProgressComponent = ({
+  progress,
+  handleSetProgress,
+  handleDragging,
+  handleStopDragging,
+}) => {
+  const inputRef = React.createRef();
+
+  useEffect(() => {
+    inputRef.current.style.background = `linear-gradient(90deg, #FFA834 0% ${progress}%, white ${progress}% 100%)`;
```

Comment:
This is wrong in so many ways 😝 

Let's avoid accessing the actual DOM.

Reply:
yeah, if somehow it is required to access the actual DOM can we make this a function and have a test for it?

I mean, since it's gonna affect the design and view and EVERYTHING in a very complicated way, let's test it.

What other ideas do you have in mind @ImAriel2D  ? 

adding a class each % and increment the class name?

any other suggestions?